### PR TITLE
Text detection opencv

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java [hunk 1]
```diff
@@ -1026,6 +1035,97 @@ static String getContentPathSafe(Content content) {
...
+        outNames.add("feature_fusion/Conv_7/Sigmoid");
+
+        List<Mat> outs = new ArrayList<>(1);
+
+        net.setInput(blob);
+        net.forward(outs, outNames);
+
+        int H = (int) (siz.height / 4); // height of those. the geometry has 4, vertically stacked maps, the score one 1
+
+        // My lord and savior : http://answers.opencv.org/question/175676/javaandroid-access-4-dim-mat-planes/
```
Comment (eugene7646) [inline Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java:1114]: Funny :)

File: Core/src/org/sleuthkit/autopsy/modules/tester/TestFactory.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    @Override
+    public DataSourceIngestModule createDataSourceIngestModule(IngestModuleIngestJobSettings settings) {
+        return new TestModule();
+    }
+    
+    class TestModule implements DataSourceIngestModule {
+
+        @Override
+        public ProcessResult process(Content dataSource, DataSourceIngestModuleProgress progressBar) {
+            File outputFile = new File("C:\\Temp\\output\\output.avi");
```
Comment (eugene7646) [inline Core/src/org/sleuthkit/autopsy/modules/tester/TestFactory.java:56]: I'm not sure I understand what are these paths and how is this class used. Looks like some leftover test code perhaps? If so, I will use it for my testing but then we should remove it.

File: Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java [hunk 3]
```diff
@@ -1026,6 +1035,97 @@ static String getContentPathSafe(Content content) {
...
+        Path eastFilePath = Paths.get("textdetection", "frozen_east_text_detection.pb");
+        File eastFile = InstalledFileLocator.getDefault().locate(eastFilePath.toString(), ImageUtils.class.getPackage().getName(), true);
+        
+        if(!eastFile.exists()) {
+            throw new CvException("Unable to find EAST model file at location " + eastFile.getAbsolutePath());
+        }
+        
+        Net net = Dnn.readNetFromTensorflow(eastFile.getAbsolutePath());
+
+        Size siz = new Size(320, 320);
```
[the remark this answers is not in the document]
Comment (kellykelly3) [reply Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java:1099]: I honestly am not completely sure.  I will try to see if I can find some information.  The Scalar value in particular is so specific and I didn't want to touch it.

File: Core/src/org/sleuthkit/autopsy/modules/tester/TestFactory.java (hunk 2 above)
Comment (kellykelly3) [reply Core/src/org/sleuthkit/autopsy/modules/tester/TestFactory.java:56]: @eugene7646 I thought I was being careful, this code should not have come into the PR.  I will remove it.

**Review by eugene7646 — APPROVED** [review]

**Pull request merged**